### ISSUE TITLE
Multiple sides

### ISSUE THREAD

**Issue by fortael** [Mango/slideout#244]
With this library possible to do menu on both sides? Because i can't. 
2 problems:
- Swipe event working wrong with 2 sides. Maybe possible to handle that event by myself? 
- Both menus are visible

Will be useful if there would be mod for <html> tag.  
`slideout-open slideout-open--left`
`slideout-open slideout-open--right`

**Comment by siddarthvader** [comment]
did you find any solution? Because I have the same issue. @fortael

**Comment by fortael** [comment]
@siddarthvader Yes...more or less. The main problem with events, that both sides are exists. If you'll try to hide second one with `display:none` then laggs on closing. 

```
.slideout-open--right .slideout-menu-left {
  visibility: hidden;
  pointer-events: none;
  z-index: -1;
}
```

Try hide second  one in this way. Then listen `beforeopen`.

`.slideout-open--right` will hide `.slideout-menu-left`. If you will open it very slow with swipe, then you will see left menu for a moment. For this i have only idea - use hammer.js to handle events by myself. 

But for now at least both works okay.

**Comment by siddarthvader** [comment]
worked like charm. Thanks.

**Comment by siddarthvader** [comment]
slideLeft.on('open', () => {
        document.getElementById('kickstart').classList.add('slideout-open--left');
        slideRight.disableTouch();
    });
    
      slideLeft.on('close', () => {
         document.getElementById('kickstart').classList.remove('slideout-open--left');
        slideRight.enableTouch();
    });
    
    
    slideRight.on('open', () => {
        document.getElementById('menu').setAttribute('data-slideout-ignore', true);
        slideLeft.disableTouch();
    });
    
    slideRight.on('close', () => {
        document.getElementById('menu').removeAttribute('data-slideout-ignore');
        slideLeft.enableTouch();
    });`


I did something like this and it worked for me like charm.
